src/btree/bt_vrfy.c

```diff
@@ -280,12 +284,31 @@ __verify_key_hs(WT_SESSION_IMPL *session, WT_ITEM *key, WT_VSTUFF *vs)
 #else
         WT_UNUSED(vs);
 #endif
-    }
 
+        /* Perform Verification of data continuity */
```

Comment:
full stop. Also what have we decided about dealing with gaps in timestamps? If all tombstones are going to be masked by cursors, how can we actually determine when a value _should_ and _shouldn't_ exist?

Reply:
Hmm that could be annoying, but I guess there's nothing we can do about that.